packages/react/src/components/CardEditor/CardEditForm/CardEditFormItems/DataSeriesFormItemModal.jsx

```diff
@@ -78,6 +78,13 @@ const propTypes = {
       })
     ),
     aggregationMethod: PropTypes.string,
+    downSampleMethods: PropTypes.arrayOf(
+      PropTypes.shape({
+        id: PropTypes.string,
+        text: PropTypes.string,
+      })
+    ),
+    downSampleMethod: PropTypes.string,
```

Comment:
should this be part of the `i18n` prop definition?

Reply:
Downsample methods follow the same structure as Aggregation methods. We handle the translation in monitor.